src/fn_call.rs

```diff
@@ -507,14 +519,20 @@ pub trait EvalContextExt<'a, 'mir, 'tcx: 'a+'mir>: crate::MiriEvalContextExt<'a,
                     Scalar::Bits { .. } => return err!(ReadBytesAsPointer),
                 };
 
-                // Figure out how large a pthread TLS key actually is. This is libc::pthread_key_t.
-                let key_type = args[0].layout.ty.builtin_deref(true)
-                                   .ok_or_else(|| EvalErrorKind::AbiViolation("Wrong signature used for pthread_key_create: First argument must be a raw pointer.".to_owned()))?.ty;
+                // Figure out how large a pthread TLS key actually is.
+                // This is `libc::pthread_key_t`.
+                let key_type =
+                    args[0].layout.ty.builtin_deref(true)
```

Comment:
use chaining block indent here and don't split up the string

Reply:
@solson That's not the convention in Rustc, but if you want it to be here, fair enough...